src/validation_workflow/yum/validation_yum.py

```diff
@@ -53,7 +54,9 @@ def installation(self) -> bool:
                 logging.info('Removed previous versions of Opensearch')
                 urllink = f"{self.args.file_path.get(project)} -o /etc/yum.repos.d/{os.path.basename(self.args.file_path.get(project))}"
                 execute(f'sudo curl -SL {urllink}', ".")
-                execute(f"sudo yum install '{project}-{self.args.version}' -y", ".")
+                execute(f"sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD={get_password(str(self.args.version))} yum install '{project}-{self.args.version}' -y", ".")
+            if self.args.allow_without_security:
+                self.args.allow_without_security = self.test_security_plugin("/usr/")
```

Comment:
If this is keep repeating can we move it to parent class?

Reply:
We might have to pass the working directory from child class only if the allow_without_security is passed by the user.

Reply:
The does not stop you from moving it to parent class?

Reply:
After talking to @Divyaasm on this.
I think she should not re-use `self.args.allow_without_security` and assign a different bool value.
Suggest just using a different var to save the output of `self.test_security_plugin("/usr/")`.